ngraph/python/tests/test_ngraph/test_ops_reshape.py

```diff
@@ -125,13 +126,31 @@ def test_gather():
         [1.0, 1.1, 1.2, 2.0, 2.1, 2.2, 3.0, 3.1, 3.2], np.float32
     ).reshape((3, 3))
     input_indices = np.array([0, 2], np.int32).reshape(1, 2)
-    input_axes = np.array([1], np.int32)
+    input_axis = np.array([1], np.int32)
 
     expected = np.array([1.0, 1.2, 2.0, 2.2, 3.0, 3.2], dtype=np.float32).reshape(
         (3, 1, 2)
     )
 
-    result = run_op_node([input_data], ng.gather, input_indices, input_axes)
+    result = run_op_node([input_data], ng.gather, input_indices, input_axis)
+    assert np.allclose(result, expected)
+
+
+@xfail_issue_49062
+def test_gather_batch_dims_1():
+
+    input_data = np.array([[1, 2, 3, 4, 5],
+                           [6, 7, 8, 9, 10]], np.float32)
+
+    input_indices = np.array([[0, 0, 4],
+                              [4, 0, 0]], np.int32)
+    input_axes = np.array([1], np.int32)
```

Comment:
nit: use `input_axes` or `input_axis` in both cases for consistency.

Also, axis can be a scalar as well. Can you test it please?